smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/rendering/serde/SerializeStructGenerator.kt

```diff
@@ -679,3 +668,10 @@ open class SerializeStructGenerator(
         return "serialize$suffix"
     }
 }
+
+fun TimestampFormatTrait.Format.toRuntimeEnum(): String = when (this) {
```

Comment:
suggestion

would be nice to link to the spec that defines this mapping for future readers.  https://awslabs.github.io/smithy/1.0/spec/core/protocol-traits.html#timestampformat-trait